src/sentry/static/sentry/app/components/charts/backendStyles.tsx

```diff
@@ -0,0 +1,72 @@
+/**
+ * This module is used to define the look and feels for charts rendered via the
+ * backend chart rendering service.
+ *
+ * Be careful what you import into this file, as it will end up being bundled
+ * into the configuration file loaded by the service
+ */
+
+import {StyleConfig, StyleDescriptor} from '@sentry-internal/chart-renderer/lib/types';
+import {EChartOption} from 'echarts/lib/echarts';
+
+import theme from 'app/utils/theme';
+
+import Grid from './components/grid';
+import Legend from './components/legend';
+import XAxis from './components/xAxis';
+import YAxis from './components/yAxis';
+import AreaSeries from './series/areaSeries';
+import {getColorPalette} from './utils/getColorPalette';
+
+/**
+ * Defines the keys which may be passed into the backend chart rendering service
+ */
+export enum BackendChartStyle {
+  SlackDiscoverTotalPeriod = 'slack:discover.totalPeriod',
```

Comment:
I noticed that we have enums in snake cases with uppercase and PascalCase  🤷‍♀️

```suggestion
  SLACK_DISCOVER_TOTAL_PERIOD = 'slack:discover.totalPeriod',
```

Reply:
Fixed